backend/migrations/versions/bfc9bdad220c_added_squad_emoji_column_removed_auth_.py

```diff
@@ -0,0 +1,28 @@
+"""Added squad emoji column, removed auth_hash column, replaced username with user as their email in Squad
```

Comment:
Is this migration accurate? In the code and here I only see that you added the squad emoji column. I don't see any changes where you removed the auth_hash column or replaced username with email

Reply:
I merged the commit message with the changes i made in the endpoint, my b i changed it to reflect only the squad emoji column